src/Avalonia.Controls/Platform/IInsetsManager.cs

```diff
@@ -14,11 +14,23 @@ public interface IInsetsManager
         /// </summary>
         bool? IsSystemBarVisible { get; set; }
 
+        /// <summary>
+        /// Gets or sets whether the window draws edge to edge. behind any visible system bars.
+        /// </summary>
+        bool DisplayEdgeToEdgePreference { get; set; }
+
+
+        [Obsolete("Use DisplayEdgeToEdgePreference")]
```

Comment:
The attribute should be moved between the XML doc and the property, or the XML doc won't be considered valid.

Reply:
Fixed.